lib/actions/merge.js

```diff
@@ -0,0 +1,52 @@
+const { Action } = require('./action')
+
+const checkIfMerged = async (context, prNumber) => {
+  let status = true
+
+  // return can be 204 or 404 only
+  try {
+    await context.github.pulls.checkIfMerged(
+      context.repo({ pull_number: prNumber })
+    )
+  } catch (err) {
+    if (err.status === 404) {
+      status = false
+    } else {
+      throw err
+    }
+  }
+
+  return status
+}
+
+class Comment extends Action {
+  constructor () {
+    super('merge')
+    this.supportedEvents = [
+      'pull_request.*'
+    ]
+  }
+
+  // there is nothing to do
+  async beforeValidate () {}
+
+  async afterValidate (context, settings, results) {
+    const prNumber = this.getPayload(context).number
+    const isMerged = await checkIfMerged(context, prNumber)
+
+    if (!isMerged) {
+      let mergeMethod = settings.merge_method ? settings.merge_method : 'merge'
```

Comment:
Should we validate valid mergeMethod? Return a warning when loading config with wrong mergeMethods?

Reply:
Any thoughts on this?

Reply:
fixed